pkg/cmd/util.go

```diff
@@ -34,15 +36,34 @@ type inputData struct {
 
 func getApplicationsFromFiles(files []string) ([]inputData, error) {
 	var appData []inputData
-
+	var data []byte
+	var err error
 	for _, file := range files {
-		data, err := ioutil.ReadFile(file)
-		if err != nil {
-			return nil, errors.Wrap(err, "file reading failed")
-		}
-		file, err := filepath.Abs(file)
-		if err != nil {
-			return nil, errors.Wrapf(err, "cannot determine the absolute file path of %q", file)
+
+		if file == "-" {
+			var buffer bytes.Buffer
+			scanner := bufio.NewScanner(os.Stdin)
+			for scanner.Scan() {
+
+				if strings.Contains(scanner.Text(), "name") && !strings.Contains(scanner.Text(), "-") && !strings.HasPrefix(scanner.Text(), " ") {
```

Comment:
hmm, Interesting solution. 
Maybe this could be a little bit more elegant if you use regexp instead of 3 tests with string ;-)

We should make sure that there are no unintentional consequences when we are using `name` for detecting that new file begun. Like for example what happens if `name` is at the end of the file? That is perfectly valid.

Reply:
even if `name` is at the end, it will detect :)
Though I will try to do it in regex way